docs/guide/getting-started.md

```diff
@@ -0,0 +1,295 @@
+
+### Install the Plugin
+
+```shell:no-line-numbers
+asdf plugin add nodejs https://github.com/asdf-vm/asdf-nodejs.git
+```
+
+## 5. Install a Version
+
+Now we have a plugin for Node.js we can install a version of the tool.
+
+We can observer which versions are available with `asdf list all nodejs` or a subset of versions with `asdf list all nodejs 14`.
+
+We will just install the `latest` available version:
+
+```shell:no-line-numbers
+asdf install nodejs latest
+```
+
+::: tip Note
+`asdf` enforces exact versions. `latest` is a helper throughout `asdf` that will resolve to the actual version number at the time of execution.
+:::
+
+## 6. Set a Version
+
+`asdf` performs a version lookup of a tool in all `.tool-versions` files from the current working directory up to the `$HOME` directory. The lookup occurs just-in-time when you execute a tool that `asdf` manages.
+
+::: warning
+Without a version listed for a tool execution of the tool will **error**. `asdf current` will show you the tool & version resolution, or absence of, from your current directory so you can observe which tools will fail to execute.
+:::
+
+### Global
+
+Global defaults are managed in `$HOME/.tool-versions`. Set a global version with:
+
+```shell:no-line-numbers
+asdf global nodejs latest
+```
+
+`$HOME/.tool-versions` will then look like:
+
+```
+nodejs 16.5.0
+```
+
+Some OSs already have tools installed that are managed by the system and not `asdf`, `python` is a common example. You need to tell `asdf` to pass the management back to the system. The [Versions reference section](/manage/versions.md) will guide you.
+
+### Local
+
+Local versions are defined in the `$PWD/.tool-versions` file (your current working directory). Usually, this will be the Git respository for a project. When in your desired directory execute:
+
+```shell:no-line-numbers
+asdf local nodejs latest
+```
+
+`$PWD/.tool-versions` will then look like:
+
+```
+nodejs 16.5.0
+```
+
+### Using Existing Tool Version Files
+
+`asdf` supports the migration from existing version files from other version managers. Eg: `.ruby-version` for the case of `rbenv`. This is supported on a per-plugin basis.
+
+[`asdf-nodejs`](https://github.com/asdf-vm/asdf-nodejs/) supports this via both `.nvmrc` and `.node-version` files. To enable this, add the following to your `asdf` configuration file `$HOME/.asdfrc`:
+
+```
+legacy_version_file = yes
+```
+
+See the [configuration](/manage/configuration.md) reference page for more config options.
+
+## Setup Complete!
+
+That completes the initial setup of `asdf` :tada: You can now manage `nodejs` versions for your project. Follow similar steps for each type of tool in your project!
+
+`asdf` has a many more commands to become familiar with, you can see them all by running `asdf --help` or `asdf`. The core of the commands are broken into three categories:
+
+- [core `asdf`](/manage/core.md)
+- [plugins](/manage/plugins.md)
+- [versions (of tools)](/manage/versions.md)
```

Comment:
for example this link no longer works on github

but does work for the site
https://github.com/asdf-vm/asdf/blob/c98d306a9ee8356a7bb7c61048decec2387f1836/docs/guide/getting-started.md#setup-complete